src/dead_letters.rs

```diff
@@ -0,0 +1,173 @@
+use async_trait::async_trait;
+use chrono::prelude::*;
+use core::fmt::Debug;
+use log::{error, info};
+use parquet::errors::ParquetError;
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+
+use crate::{deltalake_ext::*, transforms::TransformError};
+
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct DeadLetter {
+    pub base64_bytes: Option<String>,
+    pub json_string: Option<String>,
+    pub error: Option<String>,
+    pub timestamp: String,
+    pub date: String,
```

Comment:
I'm confused as to why both timestamp and date are necessary, but also why they're both stored as Strings, rather than letting serde do the serialize/deserialize for us?